### PR TITLE
[LiquidStaking]: add aptos liquid staking support

### PR DESCRIPTION
**Pull request by Milerius**
## Description

stake/unstake/withdraw ops for tortuga finance (aptos)

## How to test

Run C++ unit tests

### CODE REVIEW

File: src/Aptos/Signer.cpp [hunk 1]
```diff
@@ -183,7 +225,19 @@ Proto::SigningOutput Signer::sign(const Proto::SigningInput& input) {
...
+    auto liquidStakingFunctor = [](const Proto::LiquidStaking& liquidStakingMessage) {
+        switch (liquidStakingMessage.liquid_stake_transaction_payload_case()) {
+        case Proto::LiquidStaking::kStake:
+            return tortugaStakePayload(liquidStakingMessage.smart_contract_address(), liquidStakingMessage.stake());
+        case Proto::LiquidStaking::kUnstake:
+            return tortugaUnStakePayload(liquidStakingMessage.smart_contract_address(), liquidStakingMessage.unstake());
+        case Proto::LiquidStaking::kClaim:
+            return tortugaClaimPayload(liquidStakingMessage.smart_contract_address(), liquidStakingMessage.claim());
+        case Proto::LiquidStaking::LIQUID_STAKE_TRANSACTION_PAYLOAD_NOT_SET:
+            throw std::runtime_error("Nft message payload not set");
```
Comment (satoshiotomakan) [inline src/Aptos/Signer.cpp:237]: Can we return an empty transaction payload instead?

**Review by satoshiotomakan — APPROVED** [review]
Awesome! I have only one non-blocking comment

File: src/Aptos/Signer.cpp (hunk 1 above)
Comment (Milerius) [reply src/Aptos/Signer.cpp]: Fixed

**Review by satoshiotomakan — APPROVED** [review]
Re-approve

**Pull request merged**